docs/argocd-app-of-apps.md

```diff
@@ -26,6 +26,45 @@ we consider valuable on most clusters. As with all the charts, `infra-apps` can
 depending on your exact situation, for example on Red Hat OpenShift you would typically not deploy the bundled ingress
 component.
 
+## The `argoconfig` library chart
+
+Inspired by various `common` charts. The [`argoconfig` library chart](https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/argoconfig)
+helps us keep charts that manage Argo CD `Application` resources generic to some degree.
+
+It provides the `argoconfig.application` helper function which scaffolds an `Application` resource like so:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+# ... common.metadata
+spec:
+  project: "default"
+  source: {}
+  destination:
+    server: "https://kubernetes.default.svc"
+  syncPolicy: {}
+```
+
+Our app-of-apps charts invoke the function in their individual `Application` templates:
```

Comment:
After a nights sleep I'm now able to phrase what was bothering me yesterday. What I'm missing here is the information how to configure an application contained in an app-of-apps chart, e.g. how does the structure look like in `values.yaml` and what needs to be configured where.

This could possibly be a topic on top of the `argoconfig` section, because it would allow the reader to read on from a usage documentation to the exact implementation.

Do you understand what I'm aiming for?

Reply:
see b89b794df89b51c4c3ac0203ae45aee005aeb64c